### ISSUE TITLE
Some details?

### ISSUE THREAD

**Issue by GuHY777** [TencentYoutuResearch/PersonReID-YouReID#2]
Hello, first of all thank you for your excellent work. I am sorry that I am not very familiar with tensorflow. I would like to ask you a few specific implementation details:

1. As mentioned in the paper, conditional features are determined by two pictures. The 16x4 sampling method you use, then when calculating the subsequent conditional features, 64x63 conditional features need to be calculated?

2.Top-K, how much does K take?

Looking forward to your reply!

**Comment by poorfriend** [comment]
> Hello, first of all thank you for your excellent work. I am sorry that I am not very familiar with tensorflow. I would like to ask you a few specific implementation details:
> 
> 1. As mentioned in the paper, conditional features are determined by two pictures. The 16x4 sampling method you use, then when calculating the subsequent conditional features, 64x63 conditional features need to be calculated?
> 
> 2.Top-K, how much does K take?
> 
> Looking forward to your reply!
Thank you.    
1. Not 16x4 sampling because of the limit of gpu memory.  we  sample 4x4 in single gpu.  We need calculate 16x16 conditional features in training. you can find config in [config](https://github.com/TencentYoutuResearch/PersonReID-CACENET/blob/main/experiment/graph/cacenet.yaml) .   
2. And for Top-K, we don't  use Top-K in this implement because our further experiments show that extracting the conditional embedding on little queries with lowest top-1 similarity and ranking rest of the queries with the CACENET's individual feature already achieves CACENET's current performance. In others words,, this observation shows that the conditional embedding training branch also improves the performance of CACENET's individual feature, and the conditional feature itself is only responsible for solving the hardest cases in the test set.  In our older experiments, we set K to 100.
